testproject/Assets/Tests/Manual/NetworkAnimatorTests/AnimatedCubeController.cs

```diff
@@ -13,24 +14,19 @@ public class AnimatedCubeController : NetworkBehaviour
         private bool m_Rotate;
         private NetworkAnimator m_NetworkAnimator;
         private bool m_IsServerAuthoritative = true;
+        public bool IsServerAuthoritative => m_IsServerAuthoritative;
```

Comment:
I'm confused about this PR, it introduces this public property but I don't see it used anywhere?

Reply:
Don't be confused!  :)
This was actually pulled out of a different PR and so I think it got dropped and I just forgot to delete it.
_(Nice catch and thank you for the review!)_